pom.xml

```diff
@@ -15,7 +15,7 @@
   </licenses>
 
   <artifactId>github-api</artifactId>
-  <version>1.72.2-SNAPSHOT</version>
+  <version>1.75-SNAPSHOT</version>
```

Comment:
Ah no. It is fine because this is dev branch. This should be changed by `mvn release` process.